class-tgm-plugin-activation.php

```diff
@@ -8,7 +8,7 @@
  * or theme author for support.
  *
  * @package   TGM-Plugin-Activation
- * @version   2.5.2
+ * @version   2.6.0-RC1
  * @link      http://tgmpluginactivation.com/
  * @author    Thomas Griffin, Gary Jones, Juliette Reinders Folmer
  * @copyright Copyright (c) 2011, Thomas Griffin
```

Comment:
Is it still right and fair to claim this file is uniformly copyrighted to @thomasgriffin? There's lots of work that others have done. Thomas, would you be happy with dropping the `@copyright`, and perhaps also changing the `@author` to `The TGMPA team and contributors` or something like that?